openthread/Kconfig

```
@@ -24,4 +26,79 @@ config OPENTHREAD_LIBRARY_1_1
 
 endchoice
 
+# OpenThread configuration features sets from Nordic Semiconductor
+
+choice
+	prompt "Nordic library features set"
+	default OPENTHREAD_NORDIC_LIBRARY_NONE
+	help
+	  This option selects the Thread stack features set used for building the precompiled libraries.
```

How about adding Nordic Semiconductor in here.
It's a help text, so better to be a little more informative.
```suggestion
  Nordic Semiconductor features set selection for OpenThread stack.
```